docs/en/interfaces/formats.md

```diff
@@ -2108,35 +2108,40 @@ ClickHouse Avro format supports reading and writing [Avro data files](https://av
 
 The table below shows supported data types and how they match ClickHouse [data types](/docs/en/sql-reference/data-types/index.md) in `INSERT` and `SELECT` queries.
 
-| Avro data type `INSERT`                     | ClickHouse data type                                                                                                          | Avro data type `SELECT`       |
```

Comment:
Please, roll back this changes, please

Reply:
Since the table is formatted both using mono-spaced font & markdown, I had to update the whole table so that it is wider than before, so that it is still formatted as a nice table in mono-spaced font (when you open it e.g. in vim, or inspect the raw file). So you're okay with breaking this pretty table formatting?

Also, since I needed to insert a second asterisk reference, the one that was originally second is now third asterisk reference to keep the user focused on important things and not distract them by inserting third asterisk reference before second asterisk reference. I could possibly put the Avro Union example to the bottom of the table, but I believe it is more related to the Nullable type, since this is one of the scenarios, where we use Avro Union as well.